include/config.js

```diff
@@ -629,6 +630,12 @@ var BASE_CONFIG = {
             chain: null
         },
 
+		
+		//allow pencilblue to listen on a TCP IP port  pb.config.siteRoot+pb.config.sitePort
+		tcpip:true,
+		//allow pencilblue to create an UNIX socket
+		unixsocket:true,
```

Comment:
yes exactly, thats why I made it a variable, so you can set it to true/false, the true is just a demonstration that both can be used at the same time.